README.md

```diff
@@ -1,15 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Daniel's Porfolio
 
-> Description the project.
+> DA portfolio to showcase my skills in web programing.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- html
+- css
 
 ## Live Demo (if available)
```

Comment:
- [ ] Please edit the readme to add yourself as the author, and remove the sections that are unnecessary. Cheers 👍 

![image](https://user-images.githubusercontent.com/60618877/185596284-80f1cd6a-da17-4d19-9cc2-7f63d7aa0a1a.png)

Reply:
I have added myself as the author.